commands/search.js

```diff
@@ -152,7 +152,7 @@ class Search extends Command {
           const { allowCapture: aC2, description: desc2 } = this.formCatchDescription(userPokeBalls, energy, settings);
 
           msg.delete({ timeout: 5000 });
-          msg = await message.channel.send(`${message.author} You try to use your ${usedBall.name}, but the <:${blob.emoji_name}:${blob.emoji_id}> breaks free. ${desc2}`);
+          msg = await message.channel.send(`${message.author} You try to use your ${usedBall.name}, but the <:${blob.emoji_name}:${blob.emoji_id}>${hasBlob ? ' *(already owned)*' : ''} breaks free. ${desc2}`);
```

Comment:
Does this make it so that the user cannot catch it if they already own it?

Reply:
No.
![image](https://user-images.githubusercontent.com/8049998/35533187-0e6613c2-0580-11e8-85d5-33e631263b58.png)